lib/services/elasticsearch.js

```diff
@@ -1231,13 +1247,33 @@ class ElasticSearch extends Service {
       });
   }
 
+  _mergeDefaultMapping (index, oldMapping, newMapping) {
+    const collectionMapping = {};
+    const commonMapping = this.kuzzle.indexCache.defaultMappings[index] || this.config.commonMapping;
+
+    collectionMapping.dynamic = newMapping.dynamic || oldMapping.dynamic || this.config.dynamic;
+    collectionMapping._meta = newMapping._meta || oldMapping._meta || {};
+
+    // Inject kuzzle meta but preserve existing ones
```

Comment:
```suggestion
    // Inject kuzzle info but preserve existing ones
```

Reply:
I disagree, everywhere for are talking about Kuzzle Metadata, it will be confusing to say "Kuzzle info" here